tools/ci/src/main.rs

```diff
@@ -161,7 +161,7 @@ fn main() {
     if what_to_run.contains(Check::CFG_CHECK) {
         // Check cfg and imports
         std::env::set_var("RUSTFLAGS", "-D warnings");
-        cmd!(sh, "cargo check -Zcheck-cfg --workspace")
+        cmd!(sh, "cargo +nightly check -Zcheck-cfg --workspace")
```

Comment:
This should be its own PR :)

Reply:
I want to remove a couple of files from this PR.  What is the correct way to do that?  (In particular, this change to `tools/ci/src/main.rs` and `crates/bevy_macros_compile_fail_tests/tests/deref_mut_derive/missing_deref.fail.stderr`).

Reply:
Ok, did it.